tools/ci/test_controller.sh

```diff
@@ -69,6 +69,10 @@ export JAVA_TOOL_OPTIONS="-XX:+HeapDumpOnOutOfMemoryError"
 # some tests provide additional logs if they find this variable
 export IS_CI=true
 
+export WATCHDOG_ADDITIONAL_MONITORING_FILES="$DEBUG_FILES_OUTPUT_DIR/mvn-*.log"
```

Comment:
Wouldn't it be better to pass this variable as a parameter to `source "${HERE}/watchdog.sh"`? Of course, we would have to handle the variable in `watchdog.sh` properly, then, since it has to be accessed through `$1`. The variable `$WATCHDOG_ADDITIONAL_MONITORING_FILES` is nowhere else used.

Reply:
Somehow I consider "source" something like an "import" statement, that doesn't support passing variables.
And this seems to be a feature only available in bash: https://unix.stackexchange.com/questions/5024/passing-variables-to-a-bash-script-when-sourcing-it